mods/wool/init.lua

```diff
@@ -26,25 +20,28 @@ wool.dyes = {
 	{"dark_green", "Dark Green", "unicolor_dark_green"},
 }
 
-for _, row in ipairs(wool.dyes) do
-	local name = row[1]
-	local desc = row[2]
-	local craft_color_group = row[3]
-	-- Node Definition
-	minetest.register_node("wool:"..name, {
-		description = desc.." Wool",
-		tiles = {"wool_"..name..".png"},
+for i = 1, #wool.dyes do
```

Comment:
fixed, l hope that gets merged:
https://github.com/minetest/minetest_game/pull/1165